content/en/docs/concepts/workloads/_index.md

```diff
@@ -8,31 +8,53 @@ no_list: true
 
 {{< glossary_definition term_id="workload" length="short" >}}
 Whether your workload is a single component or several that work together, on Kubernetes you run
-it inside a set of [Pods](/docs/concepts/workloads/pods).
-In Kubernetes, a Pod represents a set of running {{< glossary_tooltip text="containers" term_id="container" >}}
+it inside a set of [pods](/docs/concepts/workloads/pods).
+In Kubernetes, a `Pod` represents a set of running {{< glossary_tooltip text="containers" term_id="container" >}}
 on your cluster.
 
-A Pod has a defined lifecycle. For example, once a Pod is running in your cluster then
-a critical failure on the {{< glossary_tooltip text="node" term_id="node" >}} where that
-Pod is running means that all the Pods on that node fail. Kubernetes treats that level
-of failure as final: you would need to create a new Pod even if the node later recovers.
+`Pods` have a [defined lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/).
+For example, once a `Pod` is running in your cluster then a critical fault on the
+{{< glossary_tooltip text="node" term_id="node" >}} where that `Pod` is running means that
+all the `Pods` on that node fail. Kubernetes treats that level of failure as final: you
+would need to create a new `Pod`, even if the node later recovers.
 
-However, to make life considerably easier, you don't need to manage each Pod directly.
-Instead, you can use _workload resources_ that manage a set of Pods on your behalf.
+However, to make life considerably easier, you don't need to manage each `Pod` directly.
+Instead, you can use _workload resources_ that manage a set of `Pods` on your behalf.
 These resources configure {{< glossary_tooltip term_id="controller" text="controllers" >}}
-that make sure the right number of the right kind of Pod are running, to match the state
+that make sure the right number of the right kind of `Pod` are running, to match the state
```

Comment:
@sftim , Can you explain the updates to the resource names? I'd like to understand if this is a change to the style guide?

Reply:
The style guide _was_ updated: https://github.com/kubernetes/website/commit/826d53c81f909b25a1f7fe3fd9c154d537c402a6

What do you think? Does this change make the page easier to follow, or harder?

Reply:
It is easier to have an explicit style rule that defines something like, "always add code formatting to API resource terms and always use the API term in the content as capitalized in the code".
It is not always clear when a common term like Pod needs to use capitalization. Often a page overuses the pod term. Some of the styling of the term Pod in the page could be removed, otherwise the page starts to look like a gameboard?
Per the style guide, the other API terms use code styling. Hopefully the added styling helps readers and does not detract from reading the page.